app/labor/cache_buster.rb

```diff
@@ -7,6 +7,17 @@ module CacheBuster
   ].freeze
 
   def self.bust(path)
+    # TODO: (Alex Smith) - It would be "nice to have" the ability to use the
+    # Fastly gem here instead of custom API calls. We'd want to keep thread
+    # safety in mind. We'll also want to consider making this modular for those
+    # who don't want to use Fastly at all.
+    #
+    # Instead of HTTP calls, we could do:
+    # fastly  = Fastly.new(api_key: ApplicationConfig["FASTLY_API_KEY"])
+    # service = Fastly::Service.new({ id: ApplicationConfig["FASTLY_SERVICE_ID"] }, fastly)
+    # fastly.purge(path)
+    #
+    # https://github.com/fastly/fastly-ruby#efficient-purging
```

Comment:
I'll create an issue for this and making the `fastly` gem usage more thread-safe after this PR is merged.